src/gtimelog/utils.py

```diff
@@ -14,9 +14,7 @@ def require_version(namespace, version):
         deb_package = "gir1.2-{namespace}-{version}".format(
             namespace=namespace.lower(), version=version)
         sys.exit("""Typelib files for {namespace}-{version} are not available.
-
 If you're on Ubuntu or another Debian-like distribution, please install
 them with
-
     sudo apt install {deb_package}
```

Comment:
Were these blank lines removed intentionally?  Why?

Reply:
Maube during the rebase... it was kind long... I'll check that after work